packages/components/src/VisuallyHidden/VisuallyHidden.tsx

```diff
@@ -0,0 +1,25 @@
+import { ReactNode, createElement, HTMLAttributes } from "react"
+import { OverrideClassName } from "~types/OverrideClassName"
+import styles from "./VisuallyHidden.module.scss"
+
+export type AllowedTags = "div" | "span"
+
+export type VisuallyHiddenProps = {
+  children: ReactNode
+  /**
+   * The HTML tag rendered by this component
+   */
+  tag?: AllowedTags
```

Comment:
Is there any reason this would even be anything other than a `span`? 🤔

Reply:
Good question. I can't see any usages in our repos that are using this prop. I'll remove it